src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -46,6 +47,8 @@ const ChartsHolder = (props: {
   const { chartId } = props;
   const projection = props.projections.primary;
 
+  const region = useRegion();
```

Comment:
Now that we have region here, can we remove `countyId`, `stateId`, and `county`?

Reply:
Eventually! That would be a big refactor, as `stateId` is used by `Compare`, the share buttons, etc - I prefer to clean up in smaller PRs to make it easier to review & ship